src/V3Undriven.cpp

```diff
@@ -381,7 +413,40 @@ class UndrivenVisitor final : public VNVisitor {
                     UINFO(9, " Full bus.  Entryp=" << cvtToHex(entryp) << endl);
                     warnAlwCombOrder(nodep);
                 }
-                entryp->drivenWhole();
+                if (entryp->isDrivenWhole() &&
+		    !m_inBBox &&
+		    !VN_IS(nodep, VarXRef) &&
+		    !nodep->underGenerate() &&
+		    !entryp->isUnderGen()) {
+			if (m_alwaysCombp &&
+                    	    (!entryp->isDrivenAlwaysCombWhole() ||
+                    	    (entryp->isDrivenAlwaysCombWhole() && m_alwaysCombp!=entryp->getAlwCombp()))) {
+                    	    nodep->v3warn(MULTIDRIVEN,
+                                  "Variable written to in always_comb also written by other process"
+                                      << " (IEEE 1800-2017 9.2.2.2): "
+                                      << nodep->prettyNameQ() << '\n'
+                                      << nodep->warnOther() << '\n'
+                                      << nodep->warnContextSecondary() << '\n'
```

Comment:
```suggestion
                                      << nodep->warnContextPrimary() << '\n'
```

First one in warning uses Primary, later ones Secondary.